### PR TITLE
ports deltastation shuttle from ss13, assigned to fland installation

### PR DESCRIPTION
**Pull request by JoeHammad1844**
## About the PR

ports the deltastation shuttle from ss13
features:
EVA area
medibot
fire axe
shotgun cabinet
sectec
telecomms server tuned to common
engi area has access to space (with fans)
## Why / Balance

the courser was too small for the map

## Media

image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:JoeHammad
- add: Fland now has its own shuttle, ported from ss13

### CODE REVIEW

File: Resources/Prototypes/Maps/fland.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@
             !type:NanotrasenNameGenerator
             prefixCreator: 'B'
         - type: StationEmergencyShuttle
-          emergencyShuttlePath: /Maps/Shuttles/emergency_courser.yml
+          emergencyShuttlePath: /Maps/Shuttles/emergency_deltanedas.yml
         - type: StationJobs
```
Comment (metalgearsloth) [inline Resources/Prototypes/Maps/fland.yml:17]: No contrib names

**Comment by deltanedas** [comment]
realops

File: Resources/Prototypes/Maps/fland.yml (hunk 1 above)
Comment (JoeHammad1844) [reply Resources/Prototypes/Maps/fland.yml:17]: sad

Comment (deltanedas) [reply Resources/Prototypes/Maps/fland.yml]: banned by webedit :trollface:

**Comment by GoodWheatley** [comment]
Have you checked to make sure the escape pods don’t clip into the shuttle when they launch?

**Comment by JoeHammad1844** [comment]
> Have you checked to make sure the escape pods don’t clip into the shuttle when they launch?

yes

**Pull request merged**